skyportal/enum_types.py

```diff
@@ -1,16 +1,44 @@
-import sqlalchemy as sa
-from enum import Enum
+import astropy.units as u
 import inspect
+import numpy as np
+import sncosmo
+from sncosmo.bandpasses import _BANDPASSES
+from sncosmo.magsystems import _MAGSYSTEMS
+import sqlalchemy as sa
 
+from enum import Enum
 from baselayer.app.env import load_env
-
+from baselayer.log import make_log
 from . import facility_apis
 
-from sncosmo.bandpasses import _BANDPASSES
-from sncosmo.magsystems import _MAGSYSTEMS
+log = make_log('enum_types')
 
 _, cfg = load_env()
 
+# load additional bandpasses into the SN comso registry
+existing_bandpasses_names = [val['name'] for val in _BANDPASSES.get_loaders_metadata()]
```

Comment:
We should make a PR to sncosmo to expose the list of bandpasses, so that we don't have to use internal attributes.

Reply:
We did. We opened this up in 2020: https://github.com/sncosmo/sncosmo/issues/268. It's...complicated.

Reply:
:sweat_smile: Got it.